eva_vcf_merge/utils.py

```diff
@@ -33,3 +34,8 @@ def write_files_to_list(files, alias, output_dir):
         for filename in files:
             handle.write(filename + "\n")
     return list_filename
+
+
+def get_safe_str(s):
+    """Return string with non-digits/letters replaced by underscore."""
+    return re.sub('[^0-9a-zA-Z]+', '_', s)
```

Comment:
What about dot `.` or other ascii characters allowed in file

Reply:
Yeah I was a bit lazy here, I'm using this both to generate Nextflow process names (which we're [requiring to be valid Python identifiers](https://github.com/EBIvariation/eva-common-pyutils/blob/master/ebi_eva_common_pyutils/nextflow/nextflow_pipeline.py#L43)) as well as the merged filename.  I can split these functionalities apart.

Reply:
Actually just switched to using an index for the Nextflow process name, since it's less important that that one be user-friendly...